pkgs/by-name/ot/otree/package.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, stdenv
+, git
```

Needed to enable building on darwin
```suggestion
, git
, darwin
```